privacyidea/static/components/token/views/token.enrolled.daypassword.html

```diff
@@ -0,0 +1,60 @@
+<div class="row">
+    <div ng-hide="form['2stepinit'] && enrolledToken.rollout_state !== 'clientwait'"
+         ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'col-sm-12' : 'col-sm-6'">
+        <img width="{{ qrCodeWidth }}"
+             ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'centered' : 'not-centered'"
+             ng-src="{{ enrolledToken.googleurl.img }}"/>
+    </div>
+    <div ng-hide="form['2stepinit']"
+         ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'col-sm-12' : 'col-sm-6'">
+        <uib-accordion close-others="oneAtATime" ng-show="show_seed">
+            <div uib-accordion-group
+                 class="panel-default"
+                 is-open="isOpen">
+                <uib-accordion-heading ng-click="isOpen=!isOpen">
+                    <div >{{ 'The OTP key' | translate }}
+                        <span class="pull-right glyphicon" aria-hidden="true"
+                           ng-class="{'glyphicon-chevron-down': isOpen, 'glyphicon-chevron-right': !isOpen}">
+                        </span>
+                    </div>
+                </uib-accordion-heading>
+                Hex: {{ enrolledToken.otpkey.value }}<br/>
+                Base32: {{ enrolledToken.otpkey.value_b32 }}
+            </div>
+        </uib-accordion>
+
+        <div class="col-sm-12" ng-hide="$state.includes('token.wizard')">
+        <p translate ng-show="enrolledToken.googleurl.value">
+            Click <a href="{{ enrolledToken.googleurl.value }}">here</a> or scan
+            the QR Code, if you
+            want to add the Token to your Google Authenticator.
```

Comment:
...to your Authenticator App.

Reply:
https://github.com/privacyidea/privacyidea/pull/3602/commits/1d33e4a5414b7c3962159450ba40a6e9f9e29488